### ISSUE TITLE
Ties across multiple notes behaving like slurs

### ISSUE THREAD

**Issue by mjwalter** [zolaemil/meiView#27]
In standard modern practice, a tie that spans multiple notes should be drawn between each pair of adjacent notes, and not just between the first and last.

Current versions of meiView render ties like slurs, with a single symbol connecting the first and last elements (screenshot from [DC0302](http://duchemin-dev.haverford.edu/piece/DC0302), Tenor, mm. 35–37):

![screen shot 2014-07-29 at 11 35 12](https://cloud.githubusercontent.com/assets/3750681/3737324/f2906a94-1735-11e4-8311-59eb98616b33.png)

[low-priority]

**Comment by zolaemil** [comment]
This is also a meitovexflow issue, but thanks for posting it nonetheless. It is because, currently the only `i` and `t` values of the `@tie` are dealt with in meitovexflow.

**Comment by mjwalter** [comment]
Right. I forget I should be putting some of these in meitovexflow. I guess it is not simple to move the issues.

**Comment by bencornu** [comment]
Is there any solution for this problem or an other way to do it?
I can't find the issue in meitovexflow github.
thx
